package-lock.json

```diff
@@ -16330,7 +16339,7 @@
     },
     "query-string": {
       "version": "5.1.1",
-      "resolved": "https://registry.npmjs.org/query-string/-/query-string-5.1.1.tgz",
+      "resolved": "http://registry.npmjs.org/query-string/-/query-string-5.1.1.tgz",
```

Comment:
should this be reverted back to `https`? not why why npm switched it to `http` 🤔